### ISSUE TITLE
Authentication/Signin Functionality

### ISSUE THREAD

**Issue by dpfernandes** [HackYourFuture-CPH/rate-my-cv#183]
## User story

**Who:** **As a** user

**What:** **I want to** be able to sign in

**Why:** **so that we can** check my profile, make reviews and upload my cv

## Acceptance criteria

Firebase for authentication with backend verification.
Check this article for implementation https://dev.to/emeka/securing-your-express-node-js-api-with-firebase-auth-4b5f

Create authentication based @senner007's video with a firebase and react example.
https://1drv.ms/v/s!AoI-8hQGtXUmnH6BnMzKNhd5uNci?e=ppJwdQ
It is using this repo :
https://github.com/senner008/Firebase-React-Express.git
so start by cloning this.
Also check out the resources for links to docs on firebase
https://github.com/HackYourFuture-CPH/React/tree/class11/resources

**Comment by arya56** [comment]
Hey @dpfernandes , In profile PR #159 the sign in procedure has been functioned.

**Comment by pdsorensen** [comment]
@ember-b-Moss i think you mentioned this is implemented, could you point me to the relevant code places? :)

**Comment by ember-b-Moss** [comment]
yes I was wrong.. authentication with firebase only is implemented with sign-up form and not sign-in.. as far as i understand..
